multinode_integration_tests/src/command.rs

```diff
@@ -63,11 +63,10 @@ impl Command {
 
     fn truncate_long_string(mut string: String) -> String {
         if string.len() <= MAX_PRINTED_STRING_SIZE {
```

Comment:
This could be further improved by reversing the test (string.len() > MAX_PRINTED_STRING_SIZE) and eliminating the empty then clause, using the else clause instead.